webapp/apps/dynamic/tests/test_behavioral.py

```diff
@@ -148,12 +148,15 @@ def test_get_not_avail_page_renders(self, start_year, start_year_is_none):
                                         UrlModel=DynamicBehaviorOutputUrl)
 
         model = unique_url.unique_inputs
-        model.raw_input_fields = None
-        model.input_fields = None
-        model.deprecated_fields = None
+        model.raw_input_fields = {}
+        model.input_fields = {}
+        model.deprecated_fields = {}
         model.tax_result = "unrenderable"
         if start_year_is_none:
             model.first_year = None
+        model.micro_sim = get_taxbrain_model(fields,
+                                             taxcalc_vers="0.14.2",
+                                             webapp_vers="1.3.0")
         model.save()
```

Comment:
You should use the microsim fields here. Otherwise the form validation will fail because `_BE_inc` will be included.

Something like this could work: 
```
        micro_sim_fields = get_post_data(start_year, _ID_BenefitSurtax_Switches=False)
        micro_sim_fields['first_year'] = start_year
        micro_sim = get_taxbrain_model(micro_sim_fields,
                                       taxcalc_vers="0.14.2",
                                       webapp_vers="1.4.0")
```